Decomposition rule (#8070)
PennyLaneAI/pennylane: doc/releases/changelog-dev.md

<h3>New features since last release</h3>

>   * A new :func:`~.change_basis_op` function and :class:`~.ops.op_math.ChangeOpBasis` were added,
>     which allow a compute-uncompute pattern (U V U†) to be represented by a single operator. A corresponding 
>     decomposition rule has been added to support controlling the pattern by simply 
>     controlling the middle (target) operator and not U or U†.
>   [(#8023)](https://github.com/PennyLaneAI/pennylane/pull/8023)
>   [(#8070)](https://github.com/PennyLaneAI/pennylane/pull/8070)

Reviewer: ```suggestion
  * A new :func:`~.change_basis_op` function and :class:`~.ops.op_math.ChangeOpBasis` were added,
    which allow a compute-uncompute pattern (U V U†) to be represented by a single operator.  
    A corresponding decomposition rule has been added to support efficiently controlling the pattern,
    in which only the central (target) operator is controlled, and not U or U†.
  [(#8023)](https://github.com/PennyLaneAI/pennylane/pull/8023)
  [(#8070)](https://github.com/PennyLaneAI/pennylane/pull/8070)
```
